app/v1/support/DesResponseMappingSupport.scala

```diff
@@ -20,10 +20,26 @@ import utils.Logging
 import v1.controllers.EndpointLogContext
 import v1.models.errors._
 import v1.models.outcomes.ResponseWrapper
+import v1.models.response.retrieveBusinessDetails.{RetrieveBusinessDetailsResponse, RetrieveBusinessDetailsResponseList}
 
 trait DesResponseMappingSupport {
   self: Logging =>
 
+  final def filterId(
+    responseWrapper: ResponseWrapper[RetrieveBusinessDetailsResponseList],
+    businessId: String
+  ): Either[ErrorWrapper, ResponseWrapper[RetrieveBusinessDetailsResponse]] = {
+    val filteredBusinesses = responseWrapper.responseData.retrieveBusinessDetailsResponse.filter {
+      BusinessDetails => businessId == BusinessDetails.businessId
```

Comment:
You shouldn't start variable names with capital letters in scenarios like this. Also, this could just be rewritten as something like `_.businessId == businessId`